wrench/src/yaml_frame_reader.rs

```diff
@@ -464,7 +464,19 @@ impl YamlFrameReader {
             return None;
         }
 
-        let file = rsrc_path(&item["image"], &self.aux_dir);
+        //rsrc_path(&item["image"], &self.aux_dir);
```

Comment:
Looks like there is a bit of old code left over here.

Reply:
that was intentional, but I don't mind removing this